.gitlab-ci.yml

```diff
@@ -163,9 +176,19 @@ convert_raspbian_raspberrypi4:
     # Set mender-image-tests submodule to correct version
     - git submodule update --init --recursive
     # Get mender-artifact for the tests
-    - curl -f -O https://mender.s3.amazonaws.com/mender-artifact/$MENDER_ARTIFACT_VERSION/linux/mender-artifact
-    - chmod ugo+x mender-artifact
-    - mv mender-artifact /usr/bin/
+    - if is_final_tag "${MENDER_ARTIFACT_VERSION}" || is_build_tag "${MENDER_ARTIFACT_VERSION}"; then
+      # Direct download from the pool
+    -   wget "https://downloads.mender.io/repos/debian/pool/main/m/mender-artifact/mender-artifact_${MENDER_ARTIFACT_VERSION}-1%2Bubuntu%2Bjammy_amd64.deb"
+    -   dpkg --install mender-artifact_${MENDER_ARTIFACT_VERSION}-1+ubuntu+jammy_amd64.deb
+    -   rm mender-artifact_${MENDER_ARTIFACT_VERSION}-1+ubuntu+jammy_amd64.deb
+    - else
+      # Assume master and install from experimental repo
+    -   curl -fLsS https://downloads.mender.io/repos/debian/gpg | apt-key add -
+    -   echo "deb [arch=$(dpkg --print-architecture)] https://downloads.mender.io/repos/debian ubuntu/jammy/experimental main" >
```

Comment:
I guess we could get `jammy` from the `release file`? But then again...

DAYP

Reply:
Which `release file` are you talking about? I used the instructions from docs.mender.io :thinking:

Reply:
It's nothing really. I was just thinking we could dynamically get the ubuntu version, instead of hard-coding it. But it's probs not worth it. nevermind

Reply:
That is something I also thought about for the mender-docs, where now we have 5 tabs to choose distro. If any, let's fix it there and eventually propagate to our CI :)